Bojzen-I-Mitten/Stort-spelprojekt
# Sound/ragdoll

* Added collision sounds upon ragdoll collision
* The collision sound volume is dependant of the force applied to the ragdoll
* Added constant factor for additional tweaking for audio clips

*** How to test ***

* In the inspector for the ragdoll script (component) for the Chad prefab,
add an audio listener to the field "Audio listener". This should be the camera for the player
as we want 3D-sound effects for the collision sound.
* Add a sound component for the chad Prefab and attach the ragdoll impact audio
clip to the audio clip field.

Note: If the chad prefab doesn't work for some reason, you can test the functionality by adding
a gameobject with a renderskin component and a ragdoll component. You can also add a toggle ragdoll component which enabled you to press space for it to ragdoll and reset.
